integration/tests/cook/util.py

```diff
@@ -1212,7 +1212,8 @@ def hosts_to_consider(cook_url, mesos_url):
     state = get_mesos_state(mesos_url)
     slaves = state['slaves']
     pool = default_pool(cook_url)
-    slaves = [s for s in slaves if s['attributes']['cook-pool'] == pool] if pool else slaves
```

Comment:
what about doing something like `s['attributes'].get('cook-pool', pool) == pool`? That would get you back on one line.

Reply:
Good call, I changed it to:

```python
s['attributes'].get('cook-pool', None) == pool
```

because we don't want slaves that don't have the attribute to be considered.